Cargo.toml

```diff
@@ -30,6 +30,7 @@ anyhow = "1.0"
 base64 = { package = "base64-compat", version = "1.0" }
 bech32 = "0.9"
 bip39 = "1.0"
+bip78 = { git = "https://github.com/dangould/rust-payjoin", rev = "8d9b7d6", features = ["sender" ] }
```

Comment:
We need this to be published in order to publish our crate.

Reply:
certainly